x-pack/plugins/graph/server/saved_objects/graph_workspace.ts

```diff
@@ -10,7 +10,8 @@ import { graphMigrations } from './migrations';
 
 export const graphWorkspace: SavedObjectsType = {
   name: 'graph-workspace',
-  namespaceType: 'single',
+  namespaceType: 'multiple-isolated',
+  convertToMultiNamespaceTypeVersion: '8.0.0',
```

Comment:
@jportner we don't need to adapt any call to use `resolve` instead of `get` for this specific SO type?

Reply:
This is done separately in this PR: https://github.com/elastic/kibana/pull/109617